### PR TITLE
feat: add color helper script

### PR DESCRIPTION
**Pull request by roosta**

### CODE REVIEW

**Comment by MindTooth** [comment]
Looking good. 😁

Vim do provide a lot of nice features when it comes to extend it, and autoload is a nice place to put stuff.  Just recently became aware of this myself.

**Comment by MindTooth** [comment]
Closes https://github.com/srcery-colors/srcery-vim/issues/49.

**Comment by roosta** [comment]
You OK with merge? review still pending :) Should I rebase the refactor commits or you think they're ok as is?

File: autoload/lightline/colorscheme/srcery.vim [hunk 1]
```diff
@@ -6,35 +6,29 @@
...
-  let s:magenta      = s:getSrcColor('SrceryMagenta')
-  let s:green        = s:getSrcColor('SrceryGreen')
-  let s:bright_green = s:getSrcColor('SrceryBrightGreen')
-  let s:xgray1       = s:getSrcColor('SrceryXgray1')
-  let s:xgray2       = s:getSrcColor('SrceryXgray2')
-  let s:xgray3       = s:getSrcColor('SrceryXgray3')
-  let s:xgray4       = s:getSrcColor('SrceryXgray4')
-  let s:xgray5       = s:getSrcColor('SrceryXgray5')
-  let s:xgray6       = s:getSrcColor('SrceryXgray6')
+  let s:black        = srcery#helper#GetColor('SrceryBlack')
```
Comment (MindTooth) [inline autoload/lightline/colorscheme/srcery.vim:11]: Nitpick, maybe sort them as the others?  Believe they follow the xterm colors code.

**Review by MindTooth — CHANGES_REQUESTED** [review]

**Comment by MindTooth** [comment]
Forgot to press send. 😝

**Comment by roosta** [comment]
I'll have it sorted later, need to get back to work :) I'm just now learning about autoload myself, always found vimscripting rather daunting, but getting used to it. Might open some doors.

File: autoload/lightline/colorscheme/srcery.vim (hunk 1 above)
Comment (roosta) [reply autoload/lightline/colorscheme/srcery.vim]: sorted

**Review by MindTooth — APPROVED** [review]
Tip top. 🤙🏻

**Pull request merged**